packages/node_modules/nav-frontend-tabs/src/tabs.tsx

```diff
@@ -42,48 +42,41 @@ class Tabs extends React.Component<TabsProps, TabsState> {
         this.setState({ aktivTab: index }, () => {
             this.pure.linkRefs[index].focus();
         });
-        this.props.onChange(e, index);
+        this.props.onChange ? this.props.onChange(e, index) : null;
```

Comment:
`this.props.onChange` er påkrevd ifølge `TabsProps`. Så da er det vel ikke nødvendig med denne sjekken?

Reply:
Nei, den var ikke nødvendig.